pyproject.toml

```diff
@@ -54,7 +54,7 @@ default_section = "THIRDPARTY"
 known_first_party = "goodconf"
 
 [tool.pytest.ini_options]
-addopts = "--cov"
+addopts = "--cov=goodconf --cov-branch"
```

Comment:
Cover only `goodconf` instead of tests as well.

Reply:
Having coverage on tests will show you if a test you wrote isn't being executed for some reason. Wouldn't that be helpful?

Reply:
Mhm good point, will push (although depending on whether features are activated via python version we might have to combine coverage output at some point :D)